auto3dseg/algorithm_templates/swinunetr/scripts/dummy_runner.py

```diff
@@ -64,7 +65,7 @@ def __init__(self, output_path, data_stats_file, device_id: int = 0):
         with open(data_stats_file) as f_data_stat:
             data_stat = yaml.full_load(f_data_stat)
 
-        pixdim = parser.get_parsed_content("transforms_train#transforms#4#pixdim")
+        pixdim = parser.get_parsed_content("transforms_train#transforms#3#pixdim")
```

Comment:
I think changes in this file are introduced because of the merging conflicts https://github.com/Project-MONAI/research-contributions/actions/runs/4869881278/jobs/8684929676